pkgs/development/python-modules/patchpy/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  more-itertools,
+  pytestCheckHook,
+}:
+
+buildPythonPackage {
+  pname = "patchpy";
+  version = "0-unstable-2024-05-26";
```

Comment:
Patchpy has releases. Why are you using an unstable version?

Reply:
https://github.com/MatthewScholefield/patchpy/releases

Reply:
https://pypi.org/project/patchpy/#history

Reply:
We can probably guess the version when the version in pyproject.toml is updated https://github.com/MatthewScholefield/patchpy/commit/e00735b6a267bd8f3ed44b18a8c0cbab5c7bd0a0

Reply:
used `2.0.4-unstable-2024-05-26`